aspnetcore/toc.md

```diff
@@ -250,6 +250,8 @@
 ## [HubContext](xref:signalr/hubcontext)
 ## [Users and groups](xref:signalr/groups)
 ## [Configuration](xref:signalr/configuration)
+## [Authentication and Authorization](xref:signalr/authn-and-authz)
```

Comment:
This file looks outdated too. Did we decide to create a new "Security" ToC node within "SignalR"?

Reply:
I haven't rebased since the ToC reorganization. I'll do that.